lib/src/models/moderator.dart

```diff
@@ -0,0 +1,51 @@
+import 'dart:convert';
+
+class Moderator {
+  final String name;
```

Comment:
Since all `Moderator`s are `Redditor`s it would be nice to parse moderator names into a `RedditorRef` object instead of just returning a `String` name.